presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java

```diff
@@ -215,6 +215,12 @@ public void printFinalInfo()
             if (stats.getSpilledBytes() > 0) {
                 reprintLine("Spilled: " + formatDataSize(bytes(stats.getSpilledBytes()), true));
             }
+
+            if (stats.getRuntimeStats() != null) {
```

Comment:
Let's add comment like other fields above to show how the printed line would look like.